[WIP] core: bind to all addresses (#11389)
tarantool/tarantool: changelogs/unreleased/gh-7152-bind-all-addresses.md

> ## bugfix/core

Author: Well, looks like it is a bugfix and a feature. See the comment about dual stack below.

## bugfix/core
* Fixed a bug when it was impossible to bind multiple interfaces with a single
  port number (gh-7152).

## feature/core
* Now the server binds all URIs matching the `listen` parameter in the box
  configuration. Previously it bound only a single URI per entry. Now providing

>   a single port number makes all interfaces be listened on that port.

Reviewer: Two changelog entries for the same feature is an overkill IMO. If we aren't planning to backport it, I'd leave only the "feature" entry. Let's ask @sergepetrenko.

Author: It is now `bugfix/core`.

> * Now the server binds all URIs matching the `listen` parameter in the box
>   configuration while it used to bind only a single URI per entry. So providing
>   a single port number results in all interfaces being listened on that port.

Reviewer 2: ```suggestion
* Now the server binds all URIs matching the `listen` parameter in the box
  configuration. Previously it bound only a single URI per entry. Now providing
  a single port number makes all interfaces be listened on that port.
```

Author: Fixed.
